content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Isolate Outliers in Monolithic Services
+kind: guide
+further_reading:
+- link: "/profiler"
+  tag: "Documentation"
+  text: "Datadog Continuous Profiler"
+- link: "/profiler/compare_profiles/"
+  tag: "Documentation"
+  text: "Comparing Profiles"
+---
+
+## Overview
+
+When investigating the performance of a monolithic application -- that is, a single service that has multiple uses -- you usually need to find which parts of the code base are using the most resources. The APM Service page showing top endpoints might be a logical first place to look, but the data there is focused on the number of requests and their duration, not the impact those requests have on compute resources available on your backend. 
+
+Instead, use the Continuous Profiler's ability to filter flame graphs by endpoint usage. This allows you to both see the top resource-consuming endpoints at a glance, as well as examine which functions are taking up the most resources for each particular endpoint.
+
+This guide describes how to use the Datadog Continuous Profiler to investigate these kinds of problems.
+
+## CPU bursts
+
+The first step in a performance investigation is to identify anomalies in resource usage over time. Consider the following graph of CPU utilization over the past hour for the service `product-recommendation`:
+
+{{< img src="profiler/guide-monolithic-outliers/1-outliers-monolith-cpu-usage.png" alt="" style="width:100%;" >}}
+
+This doesn't provide concrete pointers to the answers, but you can see some anomalous peaks in CPU usage. 
+
+Select the **Show - Avg of** dropdown (highlighted) and change the graph to show `CPU Cores for Top Endpoints` instead. This graph shows how different parts of the application contribute differently to the overall CPU utilization:
+
+{{< img src="profiler/guide-monolithic-outliers/2-outliers-monolith-cpu-top-endpoints.png" alt="" style="width:100%;" >}}
+
+
+The yellow peaks show that the `GET /store_history` endpoint has some intermittent usage corresponding to the anomalies identified above. However, the peaks might be due to differences in traffic to that endpoint. To understand if profiles can provide further insights, change the metric to `CPU - Average Time Per Call for Top Endpoints`:
+
+{{< img src="profiler/guide-monolithic-outliers/3-outliers-monolith-cpu-avg-time-per-call.png" alt="" style="width:100%;" >}}
+
+Now you can see that there is an intermittent spike in CPU utilization _for each call_ to `GET /store_history`, rather than the increase being because of more calls.
```

Comment:
```suggestion
Now you can see that there is an intermittent spike in CPU utilization where  _each call_ to `GET /store_history` takes on average 3s of CPU Time, rather than the increase being because of more calls.
```

Reply:
```suggestion
Now you can see that there is an intermittent spike in CPU utilization where _each call_ to `GET /store_history` takes on average three seconds of CPU time, rather than the increase being because of more calls.
```